Assets/MRTK/SDK/Features/UX/Scripts/AppBar/AppBar.cs

```diff
@@ -461,8 +460,7 @@ private void UpdateButtons()
 
         private void UpdateTargetObject()
         {
-            var boundsProvider = Target as IBoundsTargetProvider;
-            if (boundsProvider == null || boundsProvider.Target == null)
+            if (!(Target is IBoundsTargetProvider boundsProvider) || boundsProvider.IsNull() || boundsProvider.Target == null)
```

Comment:
Repeating this comment here

https://www.thomasclaudiushuber.com/2020/03/12/c-different-ways-to-check-for-null/ suggested using

"is not" instead of !(x is y) which I think is a bit more readable for not nested conditions.